### PR TITLE
added missing id functionality for state, updated README

### PR DESCRIPTION
**Pull request by ghjor**
@Balugrizzly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1419,14 +1411,14 @@ Assuming standard config and hosting locally.
...
+    Note: Sending the entire list of player objects back for update will fail and is currently not supported. You have to send a single player object back.
 
     - #### **How to Update The Player object/s.**
 
       1. Modify and make the desired changes to the list of of Player objects that you have received in the previous step.
       2. Push a modified single player object(Not a list) to the [ws://staging.player.bloomstudio.gg/ws/player](ws://staging.player.bloomstudio.gg/ws/player) websocket
-      3. Optional Confirm that the changes took place by looking at the next data push from the websocket.
+      3. Optional: Confirm that the changes took place by looking at the next data push from the websocket.
 
 ## Examples
```
Comment (Balugrizzly) [inline README.md:1423]: Can you explain what you are doing here?
Looks strange and as if something got deleted where it should not have

**Review by Balugrizzly — CHANGES_REQUESTED** [review]
Looks Good Apart the strange readme section which i commented on 
Can be merged after resolving that

File: README.md (hunk 1 above)
Comment (ghjor) [reply README.md:1423]: This was just meant to be the fixes of "singel" to "single" and adding a ":" after the word "Optional". Not sure why all those lines got highlighted but I didn't mean to delete anything, I'm thinking it has to do with something that got auto-formatted in my editor

Comment (ghjor) [reply README.md:1423]: Actually now that I think about it, probably resulted from removing the lines that stated excluding the ID will set the first value. For each socket in the readme I removed that line plus the newline under it, 4 sockets * 2 lines each = 8 lines which is the difference from the original readme

**Comment by ghjor** [comment]
> Looks Good Apart the strange readme section which i commented on Can be merged after resolving that

Might have went about this wrong but I just pulled the current README from the dev branch and then my only changes were removing the lines about not including the ID, no typo fixes or formatting changes. Trying to avoid merge conflicts and figured those other fixes would be better suited for another pr after this one was merged. Let me know

**Pull request merged**